packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -1081,11 +1090,11 @@ export function createProjectionNode<I>({
         /**
          * Animation
          */
-        animationProgress = 0
         animationValues?: ResolvedValues
         pendingAnimation?: Process
         currentAnimation?: AnimationPlaybackControls
         mixTargetDelta: (progress: number) => void
+        progress = 0
```

Comment:
I do feel that `animationProgress` is more descriptive, why did you go for `progress` here?

Reply:
Yeah animationProgress makes sense at a class level, I'll change it